lisa/node.py

```diff
@@ -311,6 +311,10 @@ def mark_dirty(self) -> None:
         self.log.debug("mark node to dirty")
         self._is_dirty = True
 
+    def reconnect(self) -> None:
```

Comment:
When it needs to call initialize? The initialize will be called on first command. So the code just needs to close.